scene/gui/popup_menu.cpp

```diff
@@ -308,7 +319,6 @@ void PopupMenu::gui_input(const Ref<InputEvent> &p_event) {
 					emit_signal(SNAME("id_focused"), i);
 					scroll_to_item(i);
 					control->queue_redraw();
-					set_input_as_handled();
```

Comment:
Ok sorry, this one was correct ;_;
I didn't notice that it's `gui_input` not `PROCESS`.
Same the one below.

Reply:
Re-added